drivers/uart_seds/work/assembler/implem/default/C/src/assembler.c

```diff
@@ -0,0 +1,107 @@
+/* Body file for function Assembler
+ * Generated by TASTE (kazoo/templates/skeletons/c-body/function.tmplt)
+ * You can edit this file, it will not be overwritten
+
+    !! IMPORTANT                                                        !!
+    !! When you modify your design (interface view), you must update    !!
+    !! the procedures corresponding to the provided interfaces in this  !!
+    !! file. The up-to-date signatures can be found in the .ads file.   !!
+*/
+#include "assembler.h"
+
+#include <FreeRTOS.h>
+#include <semphr.h>
+
+#include <Escaper.h>
+#include <Broker.h>
+
+__attribute__((section(".sdramMemorySection")))
+static uart_seds_asn1SccUartHwas uart;
+
+__attribute__((section(".sdramMemorySection")))
+static uint64_t sentBytes = 0;
+__attribute__((section(".sdramMemorySection")))
+static uint64_t bytesToSend = 0;
+
+__attribute__((section(".sdramMemorySection")))
+static Escaper escaper;
+#define ENCODED_PACKET_BUFFER_SIZE 100
+__attribute__((section(".sdramMemorySection")))
+static uint8_t encodedPacketBuffer[ENCODED_PACKET_BUFFER_SIZE] = {""};
+#define DECODED_PACKET_BUFFER_SIZE 100
+__attribute__((section(".sdramMemorySection")))
+static uint8_t decodedPacketBuffer[DECODED_PACKET_BUFFER_SIZE] = {""};
+
+void assembler_startup(void)
+{
+}
+
+void assembler_PI_Init
+      (const uart_seds_asn1SccInitRequestData *IN_initreqseq)
+
+{
+   Escaper_init(&escaper,
+                encodedPacketBuffer,
+                ENCODED_PACKET_BUFFER_SIZE,
+                decodedPacketBuffer,
+                DECODED_PACKET_BUFFER_SIZE);
+
+   uart_seds_asn1SccUartHwasConfig config;
+   config.mInstance = UartHwas_Instance_uartHwas_Instance_3;
```

Comment:
This should be read from the configuration.

Reply:
done